Dockerfile.sdk

```diff
@@ -108,6 +112,19 @@ RUN pip3 install --upgrade numpy pillow && \
     "tritonclient-*-manylinux1_x86_64.whl" | xargs printf -- '%s[all]' | \
     xargs pip3 install --upgrade
 
+# Install DCGM
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/datacenter-gpu-manager_${DCGM_VERSION}_amd64.deb && \
+    dpkg -i datacenter-gpu-manager_${DCGM_VERSION}_amd64.deb
+
+# Install Model Analyzer
+ARG TRITON_MODEL_ANALYZER_REPO_TAG
+RUN git clone --single-branch --depth=1 -b $TRITON_MODEL_ANALYZER_REPO_TAG \
+    https://github.com/triton-inference-server/model_analyzer.git && \
+    (export VERSION=`cat /workspace/VERSION` && cd model_analyzer && python3 setup.py install)
 
-ENV PATH //workspace/install/bin:${PATH}
+# Remove Model Analyzer Git source
+RUN rm -rf model_analyzer
+
+ENV PATH /workspace/install/bin:${PATH}
```

Comment:
With pip install, this is not needed

Reply:
This was in the Dockerfile before. I think this is because of the `perf_analyzer`.